js/json/shared/JSONNode.svelte

```diff
@@ -12,7 +12,7 @@
 
 	const dispatch = createEventDispatcher();
 	let root_element: HTMLElement;
-	let collapsed = open ? false : depth >= 3;
+	let collapsed = open ? depth >= 3 : true;
```

Comment:
I don't think this logic was correct before. If open was True, then it would be closed? It meant that the open parameter had no effect. Since the open parameter now works, I changed the default value in the python class to maintain the current behavior of having it open unless it was too deep.

Reply:
If I'm undestanding correctly, I think the original logic is correct. It's saying that if `open` is `true`, then `collapsed` is always `false`. If `open` is `false`, then it's collapsed only if `depth>=3`, which is what the docstring also says.

Reply:
It's confusing behavior imo because if you set `open=False` in the python constructor, the json is still open. I backed this out though since users haven't complained.